src/librbd/ImageCtx.h

```diff
@@ -188,6 +188,9 @@ namespace librbd {
     uint64_t mtime_update_interval;
     uint64_t atime_update_interval;
 
+    bool shared_cache_enabled;
```

Comment:
Nit: no need to cache these config settings in the ```ImageCtx``` since they aren't performance critical and can be looked up when needed

Reply:
done. i have pushed one commit which remove parent cache config information from ImageCtx.  thanks